typescript-function/tslint.yml

```diff
@@ -0,0 +1,54 @@
+defaultSeverity: error
+extends:
+    - 'tslint:recommended'
+jsRules:
+rules:
```

Comment:
again, very very verbose.
longer than the code itself

Reply:
as above,

These are literally our own rules from `nodeutils` copy pasted.